### PR TITLE
Periodic-ci-secret-bootstrap: Use private bucket

### PR DESCRIPTION
**Pull request by alvaroaleman**
Needs https://github.com/kubernetes/test-infra/pull/17601 else we produce an assignment to a nil map

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -492,8 +492,8 @@ plank:
     '*': https://prow.svc.ci.openshift.org/view/gcs/
     openshift-priv: https://deck-internal-ci.svc.ci.openshift.org/view/gcs/
     openshift/private-ci-testing: https://deck-internal-ci.svc.ci.openshift.org/view/gcs/
-  job_url_template: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/{{if
-    eq .Spec.Type "presubmit"}}pr-logs/pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}logs{{end}}{{if
+  job_url_template: https://{{if .Spec.Hidden }}deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-private/{{else}}prow.svc.ci.openshift.org/view/gcs/origin-ci-test/{{end}}
```
Comment (hongkailiu) [inline core-services/prow/02_config/_config.yaml:495]: This is awesome~!

**Comment by alvaroaleman** [comment]
/retest
Needs a test-infra bump in ci-tools so pj-rehearse picks up the upstream changes. Nikolaos did that in 3d21f96c6c2f3cf2d7d3c8d1e64ae7b7c4fceb2c

**Comment by hongkailiu** [comment]
> /retest
> Needs a test-infra bump in ci-tools so pj-rehearse picks up the upstream changes. Nikolaos did that in 3d21f96c6c2f3cf2d7d3c8d1e64ae7b7c4fceb2c

Understood. Should be working now.

**Pull request merged**